src/clj/fluree/db/ledger/json_ld.cljc

```diff
@@ -54,13 +56,14 @@
   If index in provided db is newer, updates latest index held in ledger state."
   [{:keys [state] :as ledger} branch-name db]
   (log/debug "Attempting to update ledger's db with new commit:"
-             (:alias ledger) "branch:" branch-name)
+             (:alias ledger) "branch:" branch-name "to t" (:t db))
```

Comment:
Is it confusing that the alias comes after `commit: ` in the log message?

Reply:
I'll reword this.